### DISCUSSION TITLE
Is it possible to set only a starting date without an ending date or vice versa?

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [WeblateOrg/weblate#12551] [Q&A]
Is it possible to set only a starting date without an ending date or vice versa after #11894 is implemented? The old date selection can do this.

**Comment by nijel** [comment]
You can always choose a date in the future. Before it was not consistent as some places did allow omitting start/end dates and in some it didn't.

**Reply by Geeyun-JY3** [reply]
Will you consider allowing omitting start/end dates for filtering changes? I think it is still useful. 
Can I file an issue for this?
